src/utils/OpenAI/OpenAI.ts

```diff
@@ -5,13 +5,19 @@ import {
   ReconnectInterval,
 } from "eventsource-parser";
 
+// Enabled by default. Stream option is disabled only when AZURE_OPENAI_API_STREAM_ENABLED is set to `false`.
```

Comment:
To support backward compatibility, stream option is enabled by default.
Stream option is disabled only when users set `AZURE_OPENAI_API_STREAM_ENABLED=false` explicitly.